### PR TITLE
(PUP-5571) Document the /puppet/v3/environment/:environment endpoint

### PR DESCRIPTION
**Pull request by lutter**

### CODE REVIEW

File: api/docs/http_environment.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            "consumes": [ "Http[pao_w1]", "Http[pao_w2]", "Http[pao_w3]" ],
+            "node": "agent3" }
+        }
+      }
+    }
+
+The response contains the name of the environment in the `environment` key,
+and a list of applications in that environment in the `applications`
+hash. The type/title of each application is used as the key in that hash,
+and the entry for that applications consists of a hash of the components,
```
Comment (nfagerlund) [inline api/docs/http_environment.md]: Typo: should be "application" (non-plural)

**Comment by nfagerlund** [comment]
Once the typo is fixed, this looks fine to me.

File: api/docs/http_environment.md (hunk 1 above)
Comment (lutter) [inline api/docs/http_environment.md]: Fixed

**Comment by lutter** [comment]
Made all the suggested changes - please merge

**Comment by kylog** [comment]
The travis failure can be ignored. (We were briefly pointing travis at ruby-head in the run-up to the 2.3.0 release. But ruby-head is now pointing at 2.4.0-dev ruby, which we aren't yet tracking, so that fail is expected.)

Merging.

**Pull request merged**